### PR TITLE
fix: Itemwise batch message added

### PR DESCRIPTION
**Pull request by vishdha**
TASK ID:  https://bloomstack.com/desk#Form/Task/TASK-2020-00158
New Stock Entry 2 - New Stock Entry 2

### CODE REVIEW

**Comment by neilLasrado** [comment]
@vishdha why have you done it this way?  why not fix the batch validation in Stock Ledger Entry rather than overriding it?

**Comment by vishdha** [comment]
@neilLasrado check now

File: erpnext/controllers/stock_controller.py [hunk 1]
```diff
@@ -288,6 +288,16 @@ def get_serialized_items(self):
 				tuple(item_codes))
 
 		return serialized_items
+	
+	def get_batch_items(self):
```
Comment (neilLasrado) [inline erpnext/controllers/stock_controller.py]: You wont need this function now. right?

**Review by neilLasrado — CHANGES_REQUESTED** [review]

**Comment by vishdha** [comment]
done

**Review by neilLasrado — APPROVED** [review]

**Comment by vishdha** [comment]
Develop : https://github.com/frappe/erpnext/pull/21215
hotfix: https://github.com/frappe/erpnext/pull/21214

**Pull request merged**